src/SFA.DAS.QnA.Application/Queries/Sections/GetSection/GetSectionBySectionNoHandler.cs

```diff
@@ -38,7 +41,7 @@ public async Task<HandlerResponse<Section>> Handle(GetSectionBySectionNoRequest
             return new HandlerResponse<Section>(_mapper.Map<Section>(section));
         }
 
-        private static void RemovePages(Data.Entities.Application application, ApplicationSection section)
+        private  void RemovePages(Data.Entities.Application application, ApplicationSection section)
```

Comment:
Why is this not static any more?

Reply:
cos a servie is injected.  I did play with noncreting the injected service at the top, but it goes through 2 layers of methods to be used, and is smellier than just not using a static